platform/spire/src/command.py

```diff
@@ -27,6 +27,14 @@ def fail(message: str, hint: str = None) -> None:
     raise CommandFailedException(message, hint)
 
 
+class MultipleExceptions(Exception):
+    def __init__(self, message, errs):
+        self.errs = errs
+        super().__init__('\n'.join([message] + [str(e) for e in errs]))
+        self
+        messages = [message] + [str(e) for e in errs]
+        hints = (e.hint for e in errs if hasattr(e, 'hint'))
+        super().__init__('\n'.join(messages), '\n'.join(hints))
```

Comment:
We already talked about this, but just to have a record: this looks like it's spliced wrong.